images/env-alpine/Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN apk add \
     cmake \
     intltool \
     libtool \
+    libuuid \
```

Comment:
Shouldn't it be [util-linux-dev](https://pkgs.alpinelinux.org/package/edge/main/x86/util-linux-dev)?

https://pkgs.alpinelinux.org/contents?branch=edge&name=util-linux-dev&arch=x86&repo=main